toolchain/main.py

```diff
@@ -31,6 +31,7 @@ def __print_greeting():
     ]
 
     for a, b in itertools.zip_longest(MFC_LOGO_LINES, MFC_SIDEBAR_LINES):
+        a = a if a is not None else ''
```

Comment:
nit that applies to the code that was there before: I think we can actually write `a or ''`.